toqito/state_opt/tests/test_ppt_distinguishability.py

```diff
@@ -30,21 +30,35 @@ def test_ppt_distinguishability_yyd_density_matrices():
     probs = [1 / 4, 1 / 4, 1 / 4, 1 / 4]
 
     # Min-error tests:
-    primal_res = ppt_distinguishability(states, probs=probs, dist_method="min-error", strategy=True)
-    dual_res = ppt_distinguishability(states, probs=probs, dist_method="min-error", strategy=False)
-
+    primal_res, _ = ppt_distinguishability(
+        vectors=states,
+        probs=probs,
+        subsystems=[0, 2],
+        dimensions=[2, 2, 2, 2],
+        strategy="min_error",
+        primal_dual="primal"
+    )
+    dual_res, _ = ppt_distinguishability(
+        vectors=states,
+        probs=probs,
+        subsystems=[0, 2],
+        dimensions=[2, 2, 2, 2],
+        strategy="min_error",
+        primal_dual="dual"
+    )
     np.testing.assert_equal(np.isclose(primal_res, 7 / 8, atol=0.001), True)
     np.testing.assert_equal(np.isclose(dual_res, 7 / 8, atol=0.001), True)
```

Comment:
Same as my other comment, it might be better to remove the options allowed by `np.testing`.

Reply:
Changed!